core/src/io/github/eng12020team24/project1/gamestates/AuberGame.java

```diff
@@ -11,7 +11,9 @@
 	@Override
 	public void create() {
 		batch = new SpriteBatch();
-		this.setScreen(new MenuState(this));
+		//this.setScreen(new MenuState(this));
+		//this.setScreen(new LoseState(this));
+		this.setScreen(new WinState(this));
```

Comment:
I think this should still be MenuState, otherwise you instantly win

Reply:
ah, sorry, was testing and forgot!